### ISSUE TITLE
NameError: name 'anthropic' is not defined

### ISSUE THREAD

**Issue by klimentij** [stanford-oval/storm#42]
I run 

```
python examples/run_storm_wiki_gpt.py \
    --output_dir output \
    --do-research \
    --do-generate-outline \
    --do-generate-article \
    --do-polish-article
```

and get 

```
Traceback (most recent call last):
  File "/Users/klim/storm/examples/run_storm_wiki_gpt.py", line 27, in <module>
    from lm import OpenAIModel
  File "/Users/klim/storm/./src/lm.py", line 111, in <module>
    class ClaudeModel(dspy.dsp.modules.lm.LM):
  File "/Users/klim/storm/./src/lm.py", line 196, in ClaudeModel
    (anthropic.RateLimitError,),
     ^^^^^^^^^
NameError: name 'anthropic' is not defined
```

### Update

Fixed by 

```
pip install anthropic
```

Please update requirements.txt

**Comment by Octobot222** [comment]
When Github gets back to me ill resolve the issue, i apoligize in advance!

Sent from my T-Mobile 5G Device
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Klimentij Bulygin ***@***.***>
Sent: Wednesday, May 1, 2024 7:20:29 PM
To: stanford-oval/storm ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [stanford-oval/storm] NameError: name 'anthropic' is not defined (Issue #42)


I run

python examples/run_storm_wiki_gpt.py \
    --output_dir output \
    --do-research \
    --do-generate-outline \
    --do-generate-article \
    --do-polish-article


and get

Traceback (most recent call last):
  File "/Users/klim/storm/examples/run_storm_wiki_gpt.py", line 27, in <module>
    from lm import OpenAIModel
  File "/Users/klim/storm/./src/lm.py", line 111, in <module>
    class ClaudeModel(dspy.dsp.modules.lm.LM):
  File "/Users/klim/storm/./src/lm.py", line 196, in ClaudeModel
    (anthropic.RateLimitError,),
     ^^^^^^^^^
NameError: name 'anthropic' is not defined

**Comment by shaoyijia** [comment]
Thanks! Just fixed this with commit 463634e. `anthropic` is only needed when using `ClaudeModel`.

**Comment by Octobot222** [comment]
Good deal thanks!

Sent from my T-Mobile 5G Device
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
